bin/phpunit.sh

```diff
@@ -4,4 +4,14 @@ if [[ $RUN_PHPCS == 1 || $SHOULD_DEPLOY == 1 ]]; then
 	exit
 fi
 
-if [ -f "phpunit.phar" ]; then php phpunit.phar -c phpunit.xml.dist $@; else ./vendor/bin/phpunit $@; fi;
+CURRENT_PHP_MAJOR_VERSION=$(php -r 'echo PHP_MAJOR_VERSION;')
+
+if [ $CURRENT_PHP_MAJOR_VERSION -gt 7 ]; then
```

Comment:
Should we get the hardcoded version numbers from some constant so that they're easier to update for any future releases where we might have to revisit this?

Reply:
I think it's fine to keep it as-is as this is not a frequent change. But it's worth adding a comment to clarify why we have the if-else condition here.

Reply:
This is fixed and refactored.